app/src/lib/stores/pull-request-store.ts

```diff
@@ -39,6 +38,9 @@ export class PullRequestStore {
   ): Promise<void> {
     const api = API.fromAccount(account)
 
+    this.changeActiveFetchCount(repository, c => c + 1)
```

Comment:
Shouldn't this be in the try block given this function can throw an error?

Reply:
Are you talking about the `forceUnwrap` call? Exceptions from that function _should_ be fatal as they indicate we failed some invariant.

Reply:
Yes, I was. And thanks for the clarification.